contracts/ve/FeeEstimate.vy

```diff
@@ -95,7 +95,8 @@ def estimateClaim(addr: address) -> uint256:
     max_user_epoch: uint256 = VotingEscrow(self.voting_escrow).user_point_epoch(addr)
     _start_time: uint256 = FeeDistributor(self.fee_distributor).start_time()
     _last_token_time: uint256 = FeeDistributor(self.fee_distributor).last_token_time()
-
+    _last_token_time = _last_token_time / WEEK * WEEK
```

Comment:
What's going on here? The expression "x / K * K" simplifies to "x"

Reply:
That was my original thought as well, but actually it's round down :)  This is EVM, so everything is integer.

Reply:
Thanks. I was guessing it was something like that. And I believe it's round down, at the resolution of one week.

We should put a comment above, to that effect.